packages/rich-text/src/plugins/shared/FetchingWrappedEntryCard.tsx

```diff
@@ -67,6 +69,13 @@ export function FetchingWrappedEntryCard(props: FetchingWrappedEntryCardProps) {
     getOrLoadEntry(props.entryId);
   }, [props.entryId]); // eslint-disable-line
 
+  React.useEffect(() => {
+    if (!entry) {
```

Comment:
Just wondering now... what happens if the link to the entry can't be resolved (missing permissions or the entry was deleted) ? I guess `entry === 'failed` in this case? That seems a bit confusing but ok if it works :shrug:

Reply:
yes when fetching fails `entry === 'failed'`. It is "complete" to some extent 🤔